### DISCUSSION TITLE
I forgot how to play mods.

### DISCUSSION THREAD

**Discussion by JinSakaiM** [Monika-After-Story/MonikaModDev#9101] [QA - tech support + general] [answered]
The only thing I remember is I dont have to use my current ddlc folder and I need to get a new one. I just need to know where to put the stuff inside of the folder. and also hello I haven't been on here for a while.

**Comment by JinSakaiM** [comment]
Im playing the Pink Eyes mod, please don't spoil it for me.

**Comment by Booplicate** [comment, marked as answer]
If you're asking us about other mods - we can't know. Because they could be (and probably are) made differently.
For MAS:
- [download DDLC](https://ddlc.moe/)
- [download MAS](https://monikaafterstory.com/releases.html)
- extract DDLC somewhere (like your desktop)
- extract MAS into the `game/` folder inside your DDLC install
